my_package/utils/logger.py

```diff
@@ -0,0 +1,103 @@
+from gflownet.utils.logger import Logger
+import torch
+from pathlib import Path
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import wandb
+
+
+class AL_Logger(Logger):
```

Comment:
I would suggest renaming this class to simply `Logger` and import the GFlowNet `as LoggerGFN` (or similar).

Reply:
Do you agree with this?

Reply:
yes, makes sense